src/implementations/vanilla/src/basics/flyout/tests/test-flyout.html

```diff
@@ -14,15 +15,17 @@
     </head>
     <body>
         <div class='test-container'></div>
+
         <script>
+            var i18next = Hig.i18next;
+            i18next.changeLanguage('en');
 
             function createFlyoutContent() {
                 var flyoutContent = document.createElement('div');
-                flyoutContent.textContent = "Hello, world!";
+                flyoutContent.textContent = i18next.t("general.hello_world");
```

Comment:
While it should be safe to share such a string (it is a full sentence), if this serves as example code, without more explanation this might be misleading and encourage developers to centralize too much strings (example many strings like 'new', 'none', 'view' could have different translations depending on context).  Without more explanation, and if there are no negative effects, it may be a better example to include this string in flyout section, which may lead it to be duplicated if other tests also use it?  On the other hand if other upcoming strings we need to extract are ones like 'new', 'none', 'view' and we do include multiple copies of those, and centralize "Hello, world!" that might be the most accurate example.

Reply:
The main purpose of these change is to support testing so I considered easy to reuse strings in some cases. I think we can do two things to show the best practice: 1) add a small document or link to some guideline 2) create a small implementation example aside of test example. I think the real example would be part of React implementation. So far, under "Vanilla" there are only 'test' strings.